packages/manager/modules/pci/src/projects/new/description/description.component.js

```diff
@@ -0,0 +1,9 @@
+import template from './description.html';
+
+export default {
+  template,
+  bindings: {
+    getStateLink: '<',
+    model: '<',
```

Comment:
`model` is quite vague as a naming

Reply:
It depends, `model` is a very valid name if it represents the date of the component as a whole

Reply:
:+1: For now it's just html implementation and I didn't know how to name it. I will change it during final implementation.